tests/DataFactories/ScheduledTask.php

```diff
@@ -8,17 +8,44 @@
 use MailPoet\Cron\Workers\SubscriberLinkTokens;
 use MailPoet\Cron\Workers\UnsubscribeTokens;
 use MailPoet\Cron\Workers\WooCommercePastOrders;
+use MailPoet\DI\ContainerWrapper;
+use MailPoet\Entities\ScheduledTaskEntity;
 use MailPoet\WP\Functions as WPFunctions;
 use MailPoetVendor\Carbon\Carbon;
+use MailPoetVendor\Doctrine\ORM\EntityManager;
 
 class ScheduledTask {
+  private $diContainer;
+  private $entityManager;
+
+  public function __construct() {
+    $this->diContainer = ContainerWrapper::getInstance();
+    $this->entityManager = $this->diContainer->get(EntityManager::class);
```

Comment:
I don't see any usage of `$this->diContainer` other than here in the constructor so I guess we might not need it as a property of this class. Or is here some reason why it is convenient that I'm missing?

Reply:
Good point. I pushed a new commit changing `diContainer` to be a local variable inside the constructor.